### PR TITLE
BTA-1135 - Improve the VAT incorrect credentials journey

### PR DESCRIPTION
**Pull request by gianick**

### CODE REVIEW

**Comment by gianick** [comment]
Waiting for a final check from content design regarding the URL of the new page

File: conf/messages.en [hunk 1]
```diff
@@ -44,11 +44,19 @@ site.textarea.char_limit = (Limit is {0} characters)
...
 unauthorised.novat = You haven’t added VAT to this account.
 unauthorised.correctid = Make sure you’re signed in with the correct user ID.
 
+unauthorised.title = Your VAT has not been added to this account
+unauthorised.heading = Your VAT has not been added to this account
+unauthorised.you_may_have_used_a_different_business_tax_account_ = You may have used a different business tax account in the past to manage your taxes, duties or schemes online.
+unauthorised.sign_in_to_other_account = Sign into your other account to add VAT
+unauthorised.add_vat_to_this_account = Add VAT to this account
+
+unauthorised.account_to_add_vat.title = Which account do you want to add VAT?
```
Comment (AndrewShanks) [inline conf/messages.en]: Maybe double check this wording with Lauren? I'd expect "Which account do you want to add VAT to?" or "To which account do you want to add VAT?" Similarly for the error message

File: app/views/unauthorised.scala.html [hunk 2]
```diff
@@ -24,6 +24,9 @@
   bodyClasses = None) {
 
   <h1 class="heading-large">@messages("unauthorised.heading")</h1>
-  <p>@messages("unauthorised.novat")</p>
-  <p>@messages("unauthorised.correctid")</p>
+
+  <p>@messages("unauthorised.you_may_have_used_a_different_business_tax_account_")</p>
+
+  <p><a href="@controllers.routes.UnauthorisedController.continue" class="button" role="button">@messages("site.continue")</a></p>
```
Comment (Fenwick17) [inline app/views/unauthorised.scala.html]: Can you remove the `<p>` from here, shouldnt be needed

File: app/views/components/submit_button.scala.html [hunk 3]
```diff
@@ -17,5 +17,5 @@
 @()(implicit messages: Messages)
 
 <div class="section">
-    <button id="submit" class="button">@messages("site.continue")</button>
+  <button id="submit" class="button">@messages("site.continue")</button>
```
Comment (Fenwick17) [inline app/views/components/submit_button.scala.html]: Don't need the `<div>` here

Comment (gianick) [reply app/views/components/submit_button.scala.html]: Not removing it because the 'section' class changes padding/margin of the element.

**Comment by gianick** [comment]
We also need Welsh for "Select an option"

File: conf/messages.en (hunk 1 above)
Comment (gianick) [reply conf/messages.en]: Issue was discussed and the wording will be changed as part of another story.

**Pull request merged**